### ISSUE TITLE
New Version of Docker for Mac Breaks Version Check

### ISSUE THREAD

**Issue by j-potts** [marcuslonnberg/sbt-docker#60]
Getting the following after upgrading to Docker for Mac CE (they changed the name 🙄 )...

```
java.lang.RuntimeException: Could not parse Version from 17.03.0-ce
: `.' expected but `3' found
	at sbtdocker.DockerVersion$.parseVersion(DockerVersion.scala:26)
	at sbtdocker.DockerVersion$.apply(DockerVersion.scala:20)
	at sbtdocker.DockerTag$.apply(DockerTag.scala:17)
	at sbtdocker.DockerBuild$$anonfun$buildAndTag$1.apply(DockerBuild.scala:75)
	at sbtdocker.DockerBuild$$anonfun$buildAndTag$1.apply(DockerBuild.scala:74)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at sbtdocker.DockerBuild$.buildAndTag(DockerBuild.scala:74)
	at sbtdocker.DockerBuild$.apply(DockerBuild.scala:45)
	at sbtdocker.DockerBuild$.apply(DockerBuild.scala:24)
	at sbtdocker.DockerSettings$$anonfun$baseDockerSettings$1.apply(DockerSettings.scala:17)
	at sbtdocker.DockerSettings$$anonfun$baseDockerSettings$1.apply(DockerSettings.scala:10)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[error] (*:docker) Could not parse Version from 17.03.0-ce
[error] : `.' expected but `3' found
[error] Total time: 106 s, completed Mar 2, 2017 10:16:52 AM
```

I checked out the version parsing code, I'll see if I can whip up a fix.

EDIT: I should add, this is under `sbt-docker` v 1.4.0

**Comment by markglh** [comment]
Regex regex regex

**Comment by melgene** [comment]
I've already created [pull reqest for this issue](https://github.com/marcuslonnberg/sbt-docker/pull/61)

**Comment by j-potts** [comment]
Nice one @melgene

**Comment by marcuslonnberg** [comment]
Fix available in version 1.4.1.

**Comment by felixt-cake** [comment]
Much appreciated @melgene
